pkg/agent/cniserver/pod_configuration.go

```diff
@@ -50,7 +51,10 @@ type k8sArgs struct {
 
 // setupInterface creates a veth pair: containerIface is in the container namespace and hostIface is
 // in the host namespace.
-func setupInterface(podName string, podNamespace string, ifname string, netns ns.NetNS, MTU int) (hostIface *current.Interface, containerIface *current.Interface, err error) {
+func setupInterface(
```

Comment:
Should setupInterface be one of podConfigrator's methods too like removeInterfaces?

Besides, `setupInterface` adds two interfaces like `removeInterfaces` deletes two interfaces. should we use `Interfaces` or `Interface` consistently?

Reply:
Renamed it to setupInterfaces.